feature-libs/misc/tsconfig.lib.prod.json

```diff
@@ -0,0 +1,10 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
+{
+  "extends": "./tsconfig.lib.json",
+  "compilerOptions": {
+    "declarationMap": false
```

Comment:
Should we have it configured the same as organization?

Reply:
That's the plan for the future.